pages/components/cards.js

```diff
@@ -195,3 +278,37 @@ const LikeIconImage = styled.img`
   left: 50%;
   transform: translate(-50%, -50%);
 `;
+
+const DropdownButton = styled.button`
+  background-color: transparent;
+  margin-left: 10px;
+  padding: 8px;
+  border: none;
+  cursor: pointer;
+`;
+
+const DropdownContent = styled.div`
+  display: none;
+  position: absolute;
+  min-width: 80px;
+  z-index: 1;
+  right: 0;
+`;
+
+const DropdownItem = styled.button`
+  background-color: #fff;
+  border-width: 0px;
+  padding: 12px 16px;
+  display: block;
+  cursor: pointer;
+  width: 70%;
+  margin-left: 35px;
+`;
```

Comment:
This button is called item which is misleading. Better create one styled button that can handle both of the variants here